data.json

```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2020-10-01T19:00:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Depths",
+    "difficulty": "Medium",
```

Comment:
Please fix this line, see the build log at https://github.com/swapagarwal/swag-for-dev/actions/runs/6485621033/job/17613034556?pr=1079

Reply:
okay i will do it now.